### PR TITLE
[tests] Use HAVE_FREETYPE itself instead NO_FT_FUNCS

### PR DESCRIPTION
**Pull request by ebraminio**
https://github.com/harfbuzz/harfbuzz/pull/2535#issuecomment-653933245

### CODE REVIEW

**Comment by behdad** [comment]
Thanks. In the Makefile. Am add:
```
HAVE_FREETYPE = $(HAVE_FREETYPE)
```
Then just add `HAVE_FREETYPE=$(HAVE_FREETYPE)` to the test environment.

Something like that. Or merge as is and I'll fix after.

File: test/shaping/run-tests.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def cmd(command):
 	reference = True
 	args = args[1:]
 
-no_ft_funcs = bool(int(os.getenv ('NO_FT_FUNCS', '0')))
+have_freetype = bool(int(os.getenv ('HAVE_FREETYPE', '0')))
```
Comment (behdad) [inline test/shaping/run-tests.py:18]: Don't we want it to default to 1?

**Comment by behdad** [comment]
Regarding my earlier comment about the autotools part, ignore it. Good enough as is.

File: test/shaping/run-tests.py (hunk 1 above)
Comment (ebraminio) [reply test/shaping/run-tests.py]: done

**Pull request merged**